frontend/src/data/variables/AcsHealthInsuranceProvider.ts

```diff
@@ -90,7 +90,14 @@ class AcsHealthInsuranceProvider extends VariableProvider {
 
     // Calculate totals where dataset doesn't provide it
     // TODO- this should be removed when Totals come from the Data Server
-    const total = df.pivot(["fips", "fips_name"], totalPivot).resetIndex();
+    const total = df
```

Comment:
It would be good to add a comment describing why we remove these categories in this calculation

Reply:
Done